mod_reforged/hooks/config/strings.nut

```diff
@@ -967,6 +968,19 @@ foreach (vanillaDesc in vanillaDescriptions)
 			]
 		}]
  	}),
+	RF_Decisive = ::UPD.getDescription({
+		Fluff = "There is no time to wait!",
+		Effects = [{
+			Type = ::UPD.EffectType.Passive,
+			Description = [
+				"Gain a stack of decisiveness whenever you end your turn without having used \'Wait\', up to a maximum of 3.",
```

Comment:
Minor point but I think `maximum of 3 stacks.` reads better in this case than ending at `maximum of 3.` Not a huge deal either way.